executor/builder.go

```diff
@@ -856,29 +862,16 @@ func (b *executorBuilder) buildTableReader(v *plan.PhysicalTableReader) Executor
 		ranges:    ts.Ranges,
 	}
 
-	for _, col := range v.Schema().Columns {
-		dagReq.OutputOffsets = append(dagReq.OutputOffsets, uint32(col.Index))
+	for i := range v.Schema().Columns {
+		dagReq.OutputOffsets = append(dagReq.OutputOffsets, uint32(i))
```

Comment:
For Index Look Up Reader, we don't need any data but handle from index, so we shouldn't set offsets for it